backend/seed_data/test_cases.yaml

```
@@ -5,43 +5,85 @@ test_cases:
+        outputs: 
+          - "[2,2,2]"
       - input: "[1,5,3,2]"
-        output: "[2,3,5,1]"
+        outputs: 
+          - "[2,3,5,1]"
       - input: "[]"
-        output: "[]"
+        outputs: 
+          - "[]"
 
+# Binary Trees
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)